root/components/Aliases/AliasTableBody.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+const React = require('react');
+const Frag = require('../Frag');
+const entityHREF = require('../../static/scripts/common/utility/entityHREF');
+const AliasTableRow = require('./AliasTableRow');
+const {l} = require('../../static/scripts/common/i18n');
+
+const AliasTableBody = ({aliases, allowEditing, entity}) => {
+  const aliasRows = [];
+  for (let i = 0; i < aliases.length; i++) {
+    const alias = aliases[i];
+    let actions = null;
+    if (allowEditing) {
+      actions = (
+        <Frag>
+          <a href={entityHREF(entity.entityType, entity.gid, `/alias/${alias.id}/edit`)}>
+            {l('Edit')}
+          </a>
+          {' | '}
+          <a href={entityHREF(entity.entityType, entity.gid, `/alias/${alias.id}/delete`)}>
+            {l('Remove')}
+          </a>
+        </Frag>
+      );
+    }
+    aliasRows.push(<AliasTableRow actions={actions} alias={alias} row={i % 2 ? 'even' : 'odd'} />);
+  }
+  return aliasRows;
```

Comment:
Nit: Since the name of the component is `AliasTableBody`, return a `<tbody>`?

Reply:
Yeah, I noticed that later, but I was too sleepy at the time. Will fix.